standalone/src/recording/components/recording_view/index.tsx

```diff
@@ -14,14 +14,19 @@ import {useSelector} from 'react-redux';
 import ScreenIcon from 'src/components/icons/screen_icon';
 import Timestamp from 'src/components/timestamp';
 import {callProfileImages} from 'src/recording/selectors';
-import {hostIDForCurrentCall, profilesInCurrentCallMap, screenSharingSessionForCurrentCall, sessionsInCurrentCall} from 'src/selectors';
+import {
+    hostIDForCurrentCall,
+    profilesInCurrentCallMap,
+    screenSharingSessionForCurrentCall,
+    sessionsInCurrentCall,
+} from 'src/selectors';
 
 const MaxParticipantsPerRow = 10;
 
 const RecordingView = () => {
     const {formatMessage} = useIntl();
-    const [screenPlayerNode, setScreenPlayerNode] = useState<HTMLVideoElement|null>(null);
-    const [screenStream, setScreenStream] = useState<MediaStream|null>(null);
+    const [screenPlayerNode, setScreenPlayerNode] = useState<HTMLVideoElement | null>(null);
+    const [screenStream, setScreenStream] = useState<MediaStream | null>(null);
```

Comment:
More custom linting, is this your IDE's revenge against me? :p

Reply:
Must be :)